packages/office-ui-fabric-react/src/components/Spinner/interfaces.ts

```diff
@@ -1,15 +0,0 @@
-/**
- * Deprecated at 2.0.0. Use `SpinnerSize` instead.
- * @deprecated Use `SpinnerSize` instead.
- */
-export enum SpinnerType {
```

Comment:
Removing this entire file as we have these enums in the `Spinner.types.ts` file and the exported enums are not exposed to the surface API or used anywhere else in the repo

Reply:
Seems reasonable. I confirmed this file is only accessible via absolute path import. It is _not_ exported at the component level nor are the enums used _at all_.

https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Spinner/index.ts